### PR TITLE
Revert/Fix use of generic templates for data-services docs

### PR DESCRIPTION
**Pull request by anithamuthu79**
https://jira.mesosphere.com/browse/DCOS-46318

## Urgency
- [ ] Blocker 
- [ ] High
- [ ] Medium

## Requirements
- Test all commands and procedures.
- Add [redirects](https://github.com/mesosphere/dcos-docs-site/wiki/Redirects).
- Change all affected versions (e.g. 1.7, 1.8, 1.9, 1.10, 1.11, 1.12).
- See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/wiki/Contributing).

### CODE REVIEW

File: pages/services/kafka/2.3.0-1.1.0/started/index.md [hunk 1]
```diff
@@ -8,4 +8,610 @@ model: /services/kafka/data.yml
...
+          ],
+          "status": "WAITING"
+        }
+      ],
+      "errors": [],
+      "status": "WAITING"
+    }
+-->
+
+<p class="message--note"><strong>NOTE: </strong>After a configuration update, you may see an error from Mesos-DNS; this will go away 10 seconds after the update.</p>
```
Comment (SarahStegall) [inline pages/services/kafka/2.3.0-1.1.0/started/index.md]: Glad to see this formatting! :D

**Comment by anithamuthu79** [comment]
@SarahStegall I've addressed your comments.

**Comment by imaxxs** [comment]
Other than the version - mentioned in my last comment, rest LGTM

**Comment by anithamuthu79** [comment]
> We do not need to support version 1.7, 1.8, 1.9. These should be deprecated. @vishnu2kmohan

@imaxxs Thanks for your review comments. This comment is out of scope for the list of changes made on this PR. I could definitely address this comment when we do the Service Docs re-structuring process. 

@vishnu2kmohan and @imaxxs Could you create a JIRA ticket to list the Services that are not supported (deprecated) in version 1.7, 1.8, and 1.9. This will help in our re-structure plan. 

Let me know if you'd like to chat on this. Thank You!

**Pull request merged**